### PR TITLE
Add GH Action to push to ArtifactHub

### PR DESCRIPTION
**Pull request by outscale-mdr**
Newcomers may be interested to easily install CCM through [artifacthub.io](https://artifacthub.io/)

See metadata file for [example](https://github.com/artifacthub/hub/blob/master/docs/metadata/artifacthub-repo.yml)

### CODE REVIEW

File: docs/helm.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+| ---- | ------ | --- |
+| 3DS Outscale | <support@outscale.com> |  |
+
+## Source Code
+
+* <https://github.com/outscale-dev/cloud-provider-osc/>
+
+## Requirements
+
+Kubernetes: `>=1.14.0-0`
```
Comment (outscale-vbr) [inline docs/helm.md:21]: Is it still true ?

Comment (outscale-vbr) [reply docs/helm.md:21]: Based on kubernetes api deprecation:
https://kubernetes.io/docs/reference/using-api/deprecation-guide/

Comment (outscale-vbr) [reply docs/helm.md:21]: At least 1.22.0 based on what i found here:
https://github.com/outscale-dev/cloud-provider-osc/blob/OSC-MIGRATION/deploy/k8s-osc-ccm/templates/osc-ccm.yaml
(The rbac.authorization.k8s.io/v1beta1 API version of ClusterRole, ClusterRoleBinding, Role, and RoleBinding is no longer served as of v1.22.)

Comment (outscale-vbr) [reply docs/helm.md:21]: You can also change it on csi ?

Comment (outscale-mdr) [reply docs/helm.md:21]: Good point, I'll make another PR for that

Comment (outscale-vbr) [reply docs/helm.md:21]: Ok

**Review by outscale-vbr — APPROVED** [review]

**Pull request merged**